docs/guide/http_cache.md

```diff
@@ -194,33 +194,35 @@ XKey-SoftPurge : tags, separated, with, commas
 
 To make the most of the purge option, see [Available tags](#available-tags).
 
-### Using Varnish
+## Using Varnish
 
 As eZ Platform is built on top of Symfony, it uses standard HTTP cache headers.
-By default the Symfony reverse proxy, written in PHP, is used to handle cache, but it can be easily replaced with any other reverse proxy like Varnish.
+The PHP-based Symfony reverse proxy is used by default to handle cache, but you can easily replace it with any other reverse proxy such as Varnish.
```

Comment:
```suggestion
By default the Symfony reverse proxy, written in PHP, is used to handle cache, but it can be easily replaced with any other reverse proxy like Varnish.
```